src/sls_facades.js

```diff
@@ -31,8 +31,7 @@
 /* exported filterAndSortData */
 /* exported colorForCWV */
 
-// Document properties
-const documentProperties = PropertiesService.getDocumentProperties();
+documentProperties = PropertiesService.getDocumentProperties();
```

Comment:
Not clear why this is no longer declared as const. Can you add a comment specifying where it's declared and why it's being overridden here?